src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -165,7 +164,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         logic = new LogicManager(model, storage);
 
         // Triggers the saveAddressBook method by executing an add command
-        String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
+        String addCommand = AddCommand.COMMAND_WORD + " r/person" + NAME_DESC_AMY + PHONE_DESC_AMY
```

Comment:
Can I ask what `r/person` is doing here? I am not so sure what it is doing, would it be to determine the role or if its a deafult person?

Furthermore, I think it should be a static variable

Reply:
Of course, was only used for convenience in testing. Will fix it later, lets aim to make the whole thing run first.

Reply:
r/person is here to indicate the role. It is necessary to make it a valid command. Commands without the r/ flag will be deemed invalid

Reply:
Yes. Role is an enum class consisting of PERSON, VOLUNTEER, DONOR, PARTNER. It simply serves as a "others" role, if a particular contact does not belong to any of the 3 roles we discussed initially. Also, adding this role makes testing much easier and reduces the need to modify existing test data like AMY and BOB

Reply:
Ah I see, I understand now. Sure that sounds fine 👍 

If possible, I would think that it would be better to make things clearer and improve upon code quality as when possible. Trying to circle back later on could lead to us missing things out.

I think Intellij has a function where you can search `r/person` and rename all of them to the static variable., making things easier. 

Let me know what you think.

Reply:
Sure I'll definitely change it. But would it be possible to merge this one first? If I push new commits from my side, conflict-resolving has to be done all over again as mentioned in telegram. Git removes all my changes(probably because of the reverted commit) in every resolve conflict attempt.

Reply:
Lets use the CommandTestUtil.java to abstract out " r/person" as a static constant, instead of explicitly stating it out like this.